docker/prod/Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM node:18.8.0 as node_builder
+
+COPY package.json yarn.lock ./
+RUN yarn install
+COPY assets ./assets
+COPY webpack.config.js ./
+RUN npm run build
+
+FROM php:7.4-fpm as php_builder
+
+RUN apt-get update && apt-get install -y zlib1g-dev g++ git libicu-dev zip libzip-dev libgd3 libgd-dev zip \
+    && docker-php-ext-install intl opcache pdo pdo_mysql \
+    && pecl install apcu \
+    && docker-php-ext-enable apcu \
+    && docker-php-ext-configure zip \
+    && docker-php-ext-configure gd \
+    && docker-php-ext-install zip \
+    && docker-php-ext-install gd
+
+WORKDIR /var/www/project
+
+ENV APP_ENV=prod
+
+RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
+
+RUN curl -sS https://get.symfony.com/cli/installer | bash
+RUN mv /root/.symfony5/bin/symfony /usr/local/bin/symfony
```

Comment:
Als we dit nu toch in een docker aan het plaatsen zijn. Kunnen we dan deze vage installer weghalen en `php-symfony` aan de apt-get hierboven toevoegen.

Die installer is al een keer uit het niets veranderd waardoor dingen kapot gingen namelijk